lib/android.js

```diff
@@ -483,15 +489,18 @@ function _getArtRuntimeSpec (api) {
   const endOffset = startOffset + (100 * pointerSize);
 
   const apiLevel = getAndroidApiLevel();
+  const androidVersion = getAndroidVersion();
 
   let spec = null;
 
   for (let offset = startOffset; offset !== endOffset; offset += pointerSize) {
     const value = runtime.add(offset).readPointer();
     if (value.equals(vm)) {
       let classLinkerOffset;
-      if (apiLevel >= 29) {
+      if (apiLevel == 29 && androidVersion == '10') {
```

Comment:
```suggestion
      if (apiLevel === 29 && androidVersion === '10') {
```

But checking for an exact string match on `'10'` seems like it might be very fragile across different OEM ROMs. Any better way we can detect this? Perhaps we could split on `.` and pick the first element?

Reply:
Honestly, I missed that other OEMs can have different versions for API level 29! I applied the suggested modifications and tested the changes on Nexus 6, Pixel 4, and cuttlefish emulator running AOSP master and Android 10.

Reply:
Honestly, I missed that other OEMs can have different versions for API level 29! I applied the suggested modifications and tested the changes on Nexus 6, Pixel 4, and cuttlefish emulator running AOSP master and Android 10.

Reply:
Nice, thanks! And yeah I doubt they exist right now, but wouldn't be surprised if it suddenly breaks in some update down the road :)